add documentation for RUM data collection with MAT (#31688)
DataDog/documentation: content/en/synthetics/platform/settings/_index.md

For more information, see [Connect Synthetics and APM Traces][15].

### Synthetic browser test data collection and RUM applications

To allow Datadog to collect RUM data from your browser test runs, click **Enable Synthetic RUM data collection**. If disabled, you cannot edit the RUM setting in the browser test recorder. When you are done enabling data collection, click **Save RUM Data Collection**.

Select a RUM application from the **Default Application** dropdown menu that collects browser test data. When you are done specifying a default application, click **Save RUM Data Applications**.

For more information, see [Explore RUM & Session Replay][14].

### Synthetic mobile application test data collection

> To allow Datadog to collect RUM data from your mobile application test runs, simply configure and package the RUM [iOS SDK][23] or [Android SDK][24] with your `.ipa` or `.apk` file. RUM data will automatically be linked to your test runs, giving you end-to-end observability of test runs.

Reviewer: ```suggestion
To allow Datadog to collect RUM data from your mobile application test runs, configure and package the RUM [iOS SDK][23] or [Android SDK][24] with your `.ipa` or `.apk` file. This automatically links RUM data, giving you end-to-end observability of test runs.
```

Author: > Since one of the APM integration with browser tests sections is being deleted
This shouldn't be an issue because `APM integration with browser tests` was a duplicated section.
